components/gateway/internal/auditlog/service_test.go

```diff
@@ -285,7 +285,7 @@ func TestSink_TimeoutOnWrite(t *testing.T) {
 
 	chanMsg := make(chan proxy.AuditlogMessage)
 	defer close(chanMsg)
-	sink := auditlog.NewSink(chanMsg, time.Millisecond*100)
+	sink := auditlog.NewSink(chanMsg, time.Millisecond*100, &FakeMetricCollector{})
```

Comment:
Does it make sense to have some primitive unit tests for this AuditlogCollector? Like for example some assertions that when new auditlog is received some of the AuditlogCollector's methods (Describe, Collect, SetChannelSize) are invoked with certain arguments

Reply:
sure, I can add one. Actually the meaningful methods to be called are only the `SetChannelSize` as the others are just to implement an interface required by prometheus package